sync-core/src/test/java/com/cloudant/common/CouchTestBase.java

```diff
@@ -57,7 +57,7 @@
             System.getProperty("test.couch.ignore.compaction",Boolean.FALSE.toString()));
 
     public static final Boolean IGNORE_AUTH_HEADERS = Boolean.valueOf(
-            System.getProperty("test.couch.ignore.auth.headers",Boolean.FALSE.toString()));
+            System.getProperty("test.couch.ignore.auth.headers",Boolean.TRUE.toString()));
```

Comment:
It changed, because the server needs to be set up not in admin party to be able to run the tests.